javascript/test.mjs

```diff
@@ -1,9 +1,7 @@
 import test from 'node:test';
 import assert from 'node:assert';
-import build from 'node-gyp-build';
-import process from 'node:process';
 
-const simsimd = build(process.cwd());
+import * as simsimd from "./dist/esm/simsimd.js";
```

Comment:
Is it a good idea to hard-code relative paths like this?

Reply:
Not if it were an example (which I am also thinking of adding), but for the test you want to test the generated code.